src/assets/css/sections/hero_section.css

```diff
@@ -0,0 +1,34 @@
+.hero_section {
+  display: flex;
+  flex-direction: row;
+  margin: 25px 150px 0 145px;
+}
+.hero_section_left {
+  padding: 18px;
+  margin-top: 107px;
+  margin-right: 47px;
+  width: 518px;
+  height: 331px;
+}
+.hero_section_left p {
+  font-size: var(--font-size-md);
+  font-weight: var(--fw-regular);
+  line-height: 26px;
+}
+.hero_section_button {
+  margin-top: 45px;
+}
+.hero_section_link {
+  margin-top: 60px;
+  color: var(--secondary-color);
+  font-size: var(--font-size-sm);
+  font-weight: var(--fw-bold);
+  text-decoration-line: underline;
+}
+.hero_section_link img {
+  width: 20px;
+  height: 17px;
+}
+.hero_section img {
+  margin-left: 47px;
+}
```

Comment:
The naming of file is not consistent with other. Pls check

Reply:
Thank you